pallets/loans/src/entities/loans.rs

```diff
@@ -364,6 +384,8 @@ impl<T: Config> ActiveLoan<T> {
 			}
 		}
 
+		self.repayments_on_schedule_until = T::Time::now();
```

Comment:
@mustermeiszer @hieronx should I add this line into the `repay()` method as well?

Reply:
It is atm not having any effect, right? Then I would do so.

Reply:
It has, because the `cashflow()` is computed to start from that point: https://github.com/centrifuge/centrifuge-chain/pull/1867/files#diff-637356fbd7809c17265d6b8ab519abc2b0ffa4df672cfb33a321795190f441baR254

Then I'll add it